### ISSUE TITLE
yarn not working when extentions remove

### ISSUE THREAD

**Issue by minhoyooDEV** [wallabyjs/console-ninja#26]
when i was remove ninja plugin from vscode and .vscode/extentions/{{ninja extention directory}}. yarn is not working.

Do you think it is ninja's problem?

<img width="1155" alt="스크린샷 2022-12-09 오후 2 37 32" src="https://user-images.githubusercontent.com/18508495/206632367-78b6b641-36b0-4660-9140-b659f0350b76.png">
<img width="794" alt="스크린샷 2022-12-09 오후 2 37 04" src="https://user-images.githubusercontent.com/18508495/206632373-c92f2d34-cf8c-4ddc-9743-acfe1681e6cb.png">

```
Internal Error: Cannot find module '/Users/minhoyoo/.vscode/extensions/wallabyjs.console-ninja-0.0.18/out/integrations/yarn/plugin.js'
Require stack:
- /Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs
Require stack:
- /Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at t3 (/Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs:390:10475)
    at Nf (/Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs:390:11318)
    at D (/Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs:391:21311)
    at Function.find (/Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs:391:21928)
    at async i (/Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs:446:10711)
    at async t (/Users/minhoyoo/Documents/musinsa/web/.yarn/releases/yarn-3.2.4.cjs:446:10278)
```

**Comment by NikGovorov** [comment]
https://github.com/wallabyjs/console-ninja/issues/11#issuecomment-1343740197

**Comment by NikGovorov** [comment]
We've fixed the issue in 0.0.19.

**Comment by minhoyooDEV** [comment]
Thank you Nik.
Solved it
